src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -59,10 +58,8 @@ Node *IdealLoopTree::is_loop_exit(Node *iff) const {
 }
 
 
-//=============================================================================
 
```

Comment:
May be look to remove multiply sequential empty lines too. One empty line as code separator is enough.